scipy/stats/_hypotests.py

```diff
@@ -97,9 +98,6 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     t = np.asarray(t)
     # check if x and y are valid inputs
     nx, ny = len(x), len(y)
-    if (nx < 5) or (ny < 5):
-        raise ValueError('x and y should have at least 5 elements, but len(x) '
-                         f'= {nx} and len(y) = {ny}.')
```

Comment:
We can probably keep them for for 1D input. What I'll do is try-except the function and append the error message to the warning or something. For ND I would prefer not to do this for now: no warning is emitted at all now, so emitting some sort of warning is only a step in the right direction.

Reply:
> but only adding it there and not enforcing it anymore is IMO a step back.

Wait - this makes me think that there might be a misunderstanding. These limitations are still enforced, but rather than raising a `ValueError` or return NaN and emit multiple warnings (which may or may not be informative), the function returns `nan` and emits a standard "small sample" warning that refers to the documentation for specific sample size limitations.

My first response https://github.com/scipy/scipy/pull/20694#discussion_r1607435544 still holds - for 1D input, we could append this function-specific information to the warning message without too much trouble. LMK if that's what you'd like to see.